pkg/kv/kvserver/kvflowcontrol/kvflowcontrol.go

```diff
@@ -434,7 +435,7 @@ func (s Stream) SafeFormat(p redact.SafePrinter, verb rune) {
 	if s.TenantID.IsSystem() {
 		tenantSt = "1"
 	}
-	p.Printf("t%s/s%s", tenantSt, s.StoreID.String())
+	p.Printf("t%s/s%s", redact.SafeString(tenantSt), redact.SafeString(s.StoreID.String()))
```

Comment:
instead of converting to `.String()` and then back into `SafeString`, let's just pass in `s.StoreID` directly. the `roachpb.StoreID` type already implements `redact.SafeValue`.

`roachpb.TenantID` also already implements `redact.SafeValue`, so this whole function could be:
```
  tenantID := s.TenantID
	if s.TenantID.IsSystem() {
		tenantSt = 1
	}
	p.Printf("t%s/s%s", tenantID, s.StoreID)
```

Reply:
Done for `StoreID`. Can't do the same trick with `TenantID`, since `TenantID.String()` prints the system / 1 tenant as `"system"`, and this code wants to override it to be `"1"`.